src/applications/vaos/components/InfoAlert.jsx

```diff
@@ -27,22 +29,30 @@ export default function InfoAlert({
   if (backgroundOnly) {
     return (
       <div
-        className={`usa-alert usa-alert-${status} background-color-only vads-u-display--block`}
+        className={`usa-alert usa-alert-${status} background-color-only vads-u-display--block ${classes}`}
+        role="alert"
       >
         <div className="usa-alert-body">
           {headline && (
-            <H className="usa-alert-heading vads-u-font-size--h3">{headline}</H>
+            <H
+              className="usa-alert-heading vads-u-font-size--h3"
+              role="heading"
+            >
+              {headline}
+            </H>
           )}
           <div className="usa-alert-text">{children}</div>
         </div>
       </div>
     );
   }
   return (
-    <va-alert class="vads-u-margin-top--3" status={status}>
-      <H className="vads-u-font-size--h3" slot="headline">
-        {headline}
-      </H>
+    <va-alert class={classes} status={status} role="alert">
```

Comment:
I don't think we want to add this extra role here. I think it used to be included in AlertBox, but they removed for a reason that I can't remember.